### PR TITLE
Feat: add Arabic language

### PR DESCRIPTION
**Pull request by alexeyvy**
Ticket: https://oat-sa.atlassian.net/browse/TR-1323

Add lang.rdf for Arabic to expose it to UI

Deployed on http://mca-test.playground.kitchen.it.taocloud.org:47991/

### CODE REVIEW

File: locales/ar-arb/lang.rdf [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+	xml:base="http://www.tao.lu/Ontologies/TAO.rdf#"
+	xmlns:tao="http://www.tao.lu/Ontologies/TAO.rdf#"
+>
+  <rdf:Description rdf:about="http://www.tao.lu/Ontologies/TAO.rdf#Langar-arb">
+    <rdf:type rdf:resource="http://www.tao.lu/Ontologies/TAO.rdf#Languages"/>
+    <rdfs:label xml:lang="en-US"><![CDATA[Arabic]]></rdfs:label>
+    <rdf:value><![CDATA[ar-arb]]></rdf:value>
+    <tao:LanguageUsages rdf:resource="http://www.tao.lu/Ontologies/TAO.rdf#LanguageUsageGUI"/>
+    <tao:LanguageUsages rdf:resource="http://www.tao.lu/Ontologies/TAO.rdf#LanguageUsageData"/>
+    <tao:LanguageOrientation rdf:resource="http://www.tao.lu/Ontologies/TAO.rdf#OrientationLeftToRight"/>
```
Comment (alexeyvy) [inline locales/ar-arb/lang.rdf:14]:
```suggestion
    <tao:LanguageOrientation rdf:resource="http://www.tao.lu/Ontologies/TAO.rdf#OrientationRightToLeft"/>

```

**Comment by marpesia** [comment]
It's working perfect if I add a Test taker with the Arabic language.

But I have got this issue if I try to save the item on Arabic
image

**Review by Abrahim111s — APPROVED** [review]

**Review by Abrahim111s — CHANGES_REQUESTED** [review]
locales/ar-arb/lang.rdf

**Comment by Abrahim111s** [comment]
@

**Review by wazelin — CHANGES_REQUESTED** [review]
Please update the commit message to have a lowercase feat prefix?
The uppercase one is not a conventional one, and won't be taken into account when generating the extension's version.

**Comment by alexeyvy** [comment]
@wazelin fixed, thanks

**Comment by alexeyvy** [comment]
Sorry, @Abrahim111s I didn't get your comment, can you explain it, please?

**Review by wazelin — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful
- [x] Pull request's target is not `master`

**Review by emgolubev — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful
- [x] Pull request's target is not `master`

**Pull request merged**